src/wmnds/patterns/cookies/cookies-manager/_cookies-manager.njk

```diff
@@ -47,7 +69,7 @@
       {{
         wmndsButton({
           classses: "wmnds-cookies-manager__save-preferences wmnds-col-1 wmnds-col-sm-1 wmnds-col-md-12-24",
-          contentText: "Save Preferences",
+          contentText: "Save preferences",
           role: "submit"
         })
       }}
```

Comment:
Update the 'visit our policies page' link to go to : https://www.tfwm.org.uk/policies/privacy-and-cookies-policy/